molo/core/migrations/0006_auto_20181009_1021.py

```diff
@@ -1,24 +0,0 @@
-# -*- coding: utf-8 -*-
-# Generated by Django 1.11.16 on 2018-10-09 10:21
-from __future__ import unicode_literals
-
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ('core', '0005_auto_20180906_0915'),
-    ]
-
-    operations = [
-        migrations.AlterModelOptions(
-            name='articlepage',
-            options={'ordering': ('-latest_revision_created_at',), 'verbose_name': 'Article'},
```

Comment:
I noticed that this is the build that made travis pass, '-latest_revision_created_at'  should still make the tests pass if the articles were not edited

Reply:
The previous commit is the one that made the tests pass. but this commit was made before the travis run for that one started so it just didn't run the tests for that commit and only did them for this one. If we were to restart the build for that commit I'd expect it to pass.
This commit is just to remove the migrations that shouldn't be here.
(also the migrations aren't run in the tests for molo.core so this commit makes 0 difference to the tests)